### ISSUE TITLE
Create a pathway through training

### ISSUE THREAD

**Issue by LauraHilliger** [mozilla/school-of-webmaking#64]
so it's easy to understand what to do and where to go.

**Comment by mozzadrella** [comment]
Hey @LauraHilliger do you envision this on the front page? I like the graphic you showed today.

**Comment by LauraHilliger** [comment]
I think we can link to something from the start page, or embed it there. Depends on the solution, I think. 

I had a play, but don't really like it. Too wordy, I think. https://laura.makes.org/thimble/how-to-participate-in-webmaker-trainings
After playing with this (for admittedly too long), I feel like a clean up of theinfographic might be the way to go. I did a search for something similar, but didn't find anything good. 

For reference, I really like this "how to" (which i had to access via way way back machine...)
https://web.archive.org/web/20120523162636/http://gidsy.com/handbooks/making-the-perfect-listing/

**Comment by LauraHilliger** [comment]
@mozzadrella suggested a video WHICH IS BRILLIANT. Just writing it down so we don't forget...

**Comment by LauraHilliger** [comment]
we can make a video sometime in the future, but for now this is solved with: https://laura.makes.org/thimble/LTE5MzI3ODc3MTI=/how-to-participate-in-webmaker-training
